### DISCUSSION TITLE
The default terriajs maps require license or not ?

### DISCUSSION THREAD

**Discussion by malnajdi** [TerriaJS/terriajs#5251] [Q&A] [answered]
Greetings Team, 

I wanted to ask if Positron (light) & Dark Matter require license to be able to publish it for production use ?

**Comment by nf-s** [comment, marked as answer]
Hi @malnajdi 

Those basemaps are generated and hosted by [Carto](https://carto.com/).  
Information about their Basemaps can be found here - https://carto.com/basemaps/  
and terms of use can be found here https://drive.google.com/file/d/1P7bhSE-N9iegI398QYDjKeVhnbS7-Ilk/view

For commercial projects, I believe you will need a commercial API key

Thanks!

**Reply by malnajdi** [reply]
> Hi @malnajdi
> 
> Those basemaps are generated and hosted by [Carto](https://carto.com/).
> Information about their Basemaps can be found here - https://carto.com/basemaps/
> and terms of use can be found here https://drive.google.com/file/d/1P7bhSE-N9iegI398QYDjKeVhnbS7-Ilk/view
> 
> For commercial projects, I believe you will need a commercial API key
> 
> Thanks!

Thanks a lot
